src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java

```diff
@@ -241,15 +245,88 @@ private void ensureWithinRange(@NotNull Position position) throws TileOutOfRange
             throw new TileOutOfRangeException(position, xRange, yRange);
     }
 
+    private void setGameRunningFalse(){
```

Comment:
just an idea, how about endGame() instead of seGameRunningFalse()? (feel free to ignore this comment)

Reply:
yea thats even better